src/config.ts

```diff
@@ -46,7 +47,8 @@ export const defaultRuleConfig: ConditionConfig = {
     NONE: 0
   },
   blockingLabels: [],
-  requiredLabels: []
+  requiredLabels: [],
+  blockingTitleRegex: 'wip'
```

Comment:
I'm not sure whether this should be the default. I'm thinking this should always be set by the repository.

Making `blockingTitleRegex` nullable and setting the default to `null` would force the user of the bot to choose which regex will block and will not run into implicit surprises.